packages/epetra/src/Epetra_C_wrappers.h

```diff
@@ -55,8 +55,8 @@ typedef long long  * EPETRA_LONG_LONG;
 
 #ifdef EPETRA_ADDRESS64BIT
 
-typedef long int EPETRA_OBJECT_PTR;
-typedef long int & EPETRA_OBJECT_REF;
+typedef long long int EPETRA_OBJECT_PTR;
+typedef long long int & EPETRA_OBJECT_REF;
```

Comment:
This seems reasonable.  C++11 added official support for `long long` int and it is guaranteed to be at least 64 bits.  So I would think this is the right change to make to achieve that.